src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/DescriptionAddendumRequest.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.dto.request
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+@Schema(
+  description = "Request to add an addendum to the description of the incident report",
+  accessMode = Schema.AccessMode.READ_ONLY,
+)
+@JsonInclude(JsonInclude.Include.ALWAYS)
+class DescriptionAddendumRequest(
```

Comment:
I prefer an explicit request object for API

Reply:
i agree. it's what we do for the other "related objects"